page/renderer/ComponentUtils.js

```diff
@@ -0,0 +1,67 @@
+export default class ComponentUtils {
+  /**
+   * Initializes the registry used for managing component utils.
+   *
+   * @param {ObjectContainer} oc The application's dependency injector - the
+   *        object container.
+   */
+  constructor(oc) {
+    /**
+     * The application's dependency injector - the object container.
+     *
+     * @type {ObjectContainer}
+     */
+    this._oc = oc;
+
+    /**
+     * Map of registered utilities.
+     *
+     * @type {Object<string, Object>}
+     */
+    this._utilities = {};
+  }
+
+  /**
+   * Registers single utility class or multiple classes in alias->class mapping.
+   *
+   * @param {function(new: Object)|Object<string, function(new: Object)>} componentUtilityClass
```

Comment:
The componentUtilityClass can be https://github.com/seznam/IMA.js-core/blob/master/ObjectContainer.js#L342 same as ObjectContainer.get